.github/workflows/runE2ETest.yml

```diff
@@ -48,11 +58,6 @@ jobs:
           repository: forcedotcom/salesforcedx-vscode-automation-tests
           path: salesforcedx-vscode-automation-tests
           ref: ${{ inputs.automationBranch }}
-      - name: Install Extension Dependencies
```

Comment:
why is this not needed anymore?

Reply:
Because we want the tests to run against the vsixes and not the code, so we actually don't need the code to compile. We'll discuss this further at [standup tomorrow](https://salesforce.quip.com/qaxAA3f9ilPr)